akka-edge-docs/src/main/paradox/guide/2-drone-location-to-cloud-service.md

```diff
@@ -240,8 +240,12 @@ Java
 
 The complete sample can be downloaded from GitHub, but note that it also includes the next steps of the guide:
 
-* Java: https://github.com/akka/akka-projection/tree/main/samples/grpc/restaurant-drone-deliveries-service-java
-* Scala: https://github.com/akka/akka-projection/tree/main/samples/grpc/restaurant-drone-deliveries-service-scala
+* Scala
+    * [local-drone-control-service.zip](../attachments/local-drone-control-service-scala.zip)
+    * [restaurant-drone-deliveries-service.zip](../attachments/restaurant-drone-deliveries-service-scala.zip)
+* Java
+    * [local-drone-control-service.zip](../attachments/local-drone-control-service-java.zip)
+    * [restaurant-drone-deliveries-service.zip](../attachments/restaurant-drone-deliveries-service-java.zip)
```

Comment:
no, both of them are included in same zip, which is named drone-scala.zip and drone-java.zip